src/subscription_manager/cli.py

```diff
@@ -171,16 +180,16 @@ def _find_best_match(self, args):
 
         return cmd
 
-    def main(self):
-        cmd = self._find_best_match(sys.argv)
+    def main(self) -> Optional[int]:
```

Comment:
hmm this really is `int` -- the problem is that the `try..except` block at the end of it calls `print()` rather than using `sys.exit()`; this "None" return value bubbles up
- ... to `ManagerCLI.main()`
- ... to the `src/subscription_manager/scripts/subscription_manager.py` script in its `main()` function
- ... to the `src/subscription_manager/scripts/subscription_manager.py` script in its `if __name__ == "__main__"` block, which does a sad `main() or 0`...

I think it would be nice to fix/simplify this in another PR